opm/simulators/wells/BlackoilWellModelGeneric.cpp

```diff
@@ -1576,7 +1576,7 @@ calculateEfficiencyFactors(const int reportStepIdx)
     for (auto& well : well_container_generic_) {
         const Well& wellEcl = well->wellEcl();
         Scalar well_efficiency_factor = wellEcl.getEfficiencyFactor() *
-                                        wellState()[well->name()].efficiency_scaling_factor;
+                                        wellState().getGlobalEfficiencyScalingFactor(well->name());
```

Comment:
At other places there seems to be a check that the well is owned around this instead
```
if (this->well_state_.wellIsOwned(well.indexOfWell(), well.name())) {
  ...
```
Wouldn't this work here and the other places, too?

Reply:
Nope, it won't. (I was tricked by accumlate... below).
For distributed wells, we need to do this on all processes that have cells of a well.

Reply:
@akva2 I think this is where it broke and what @vkip is fixing.

Reply:
Not sure I follow - as commented below, both network and gas lift calculations need to get the efficiency factor for off process wells.

Reply:
I don't think this block is actually where the real fix is situated. This loops over the well_container_generic_ and that only has entries for local active wells. The real reason for the change are the two call sites in GasLiftGroupInfo.

Reply:
and this one: https://github.com/OPM/opm-simulators/blob/ee972dcbe2187c1051773415dda47f37f072f2b0/opm/simulators/wells/WellGroupHelpers.cpp#L875-L876

Reply:
indeed.